website/docusaurus.config.ts

```diff
@@ -211,6 +223,10 @@ const config: Config = {
               label: 'Getting started',
               to: '/docs/getting-started/install',
             },
+            {
+              label: 'Glossary',
```

Comment:
I wouldn't put the Glossary below docs, but rather in the top navigation bar below resources

Reply:
Got it! Now it's under the `Resources` tab